teuthology/orchestra/cluster.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, remotes=None):
         """
         self.remotes = {}
         if remotes is not None:
-            for remote, roles in remotes:
+            for remote, roles in remotes.iteritems():
```

Comment:
I don't think you are fixing a grammatical error or typo here. Please provide a clear description, title and Commit message.